hologram_dashboard/README.md

```diff
@@ -18,10 +18,12 @@ Follow the following steps to run the server
 ## Extension
 Follow the following steps to upload the extension onto chrome
 
-    1.  Navigate to chrome://extensions from a chrome browser
-    2.  Enable developer's mode by clicking the developer mode option on the
+    1.  Build the angular front-end code by running ng build from the extension 
```

Comment:
nit: `ng build`

Reply:
I meant add a quote `` for the ng build. But it seems github fails to render ==.